arctic/store/version_store.py

```diff
@@ -360,6 +360,27 @@ def _get_info(self, symbol, as_of=None):
         else:
             return """Handler: %s\n\nVersion document:\n%s""" % (handler.__class__.__name__, pprint.pformat(_version))
 
+    def get_data_info(self, symbol):
```

Comment:
Looks like the only other parameter is allow_secondary - which is a bit of an implementation detail... Symbol and as_of are the key ones I think.